### PR TITLE
Neural network 2

### PR DESCRIPTION
**Pull request by salopge**
- 마지막 번역 이후로 너무 미뤘네요
- 정규화까지 초벌 번역 하였습니다
- Loss function 파트 주말까지 마무리 예정입니다
- 해석 안되는 부분 정리해서 따로 질문 올릴예정입니다

### CODE REVIEW

**Comment by myungsub** [comment]
네 업데이트 하였습니다. 수고하셨습니다!ㅎㅎ
